dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -26,6 +26,9 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(shell);
 
+BOOL IsDriveFloppyA(LPCSTR pszDriveRoot);
+BOOL IsDriveFloppyW(LPCWSTR pszDriveRoot);
```

Comment:
maybe `extern BOOL ....` to underline the fact both these functions are NOT defined in this compilation unit/file.